StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedPaymentElement+Internal.swift

```diff
@@ -543,6 +549,17 @@ extension EmbeddedPaymentElement {
             stpAssertionFailure("3DS2 was triggered unexpectedly")
         })
     }
+
+    static func validateRowSelectionConfiguration(configuration: Configuration) throws {
+        // Fail init if the merchant is using immediateAction and confirm form sheet action along w/ either a Customer or Apple Pay configuration
+        if case .immediateAction = configuration.rowSelectionBehavior,
+           case .confirm = configuration.formSheetAction,
+            configuration.applePay != nil || configuration.customer != nil {
```

Comment:
how about:
```
        if case .immediateAction = configuration.rowSelectionBehavior {
            guard case .continue = configuration.formSheetAction,
                  configuration.customer == nil,
                  configuration.applePay == nil else {
                throw PaymentSheetError.integrationError(nonPIIDebugDescription: "Using .immediateAction with .confirm form sheet action is not supported when Apple Pay or a customer configuration is provided. Use .default row selection behavior or disable Apple Pay and saved payment methods.")
            }
    ```

Reply:
I think this has a similar issue, but in reverse now. Merchants are allowed to use the `confirm` form sheet action, so long as they aren't using a Customer or Apple Pay.